Source/Scene/Model.js

```diff
@@ -3009,7 +3012,8 @@ define([
             if (!defined(clippingPlanes)) {
                 return Matrix4.IDENTITY;
             }
-            return Matrix4.multiply(model._modelViewMatrix, clippingPlanes.modelMatrix, scratchClippingPlaneMatrix);
+            var clippingPlanesModelMatrix = Matrix4.multiply(model._clippingPlaneModelMatrix, clippingPlanes.modelMatrix, scratchClippingPlaneMatrix);
```

Comment:
`modelMatrix` already incorporates `computedTransform`, so this might be double applying it and leading to unexpected results.